.vscode/settings.json

```diff
@@ -1,24 +1,24 @@
 {
-  "editor.insertSpaces": true,
-  "editor.tabSize": 2,
-  "editor.rulers" : [100],
-  "editor.acceptSuggestionOnEnter": "off",
-  "files.encoding": "utf8",
-  "files.eol": "\n",
-  "files.insertFinalNewline": true,
-  "files.trimFinalNewlines": true,
-  "files.trimTrailingWhitespace": true,
-  "githubPullRequests.ignoredPullRequestBranches": [
-    "master"
-  ],
-  "git.ignoreLimitWarning": true,
-  "git.branchProtection": [
-    "master",
-    "stable",
-    "staging"
-  ],
-  "files.readonlyInclude": {
-    "**/.elan/**": true,
-    "**/.lake/**": true
-  }
-}
+    "editor.insertSpaces": true,
```

Comment:
This seems like a mistake.

Reply:
Yes, that was a mistake, it has been corrected.